lib/convertion/converter.rb

```diff
@@ -18,15 +17,19 @@ def initialize
     @version_ds = ds_version
   end
 
+  def file_helper
+    @file_helper ||= OnlyofficeFileHelper::FileHelper
```

Comment:
I think this will be confusing since construction like this we're using in cases where there is a object instance, like

```
def file_helper
    @file_helper ||= OnlyofficeFileHelper::FileHelper.new
end
```

But in that case you've created method to call class, not class instance
Actually I've never known that this construction worked )

you can begin the class with something like 

```
class Converter
  include OnlyofficeFileHelper
...

def get_file_paths_list(path)
  FileHelper.list_file_in_directory
```  
I'm not 100% sure should be `include`, or may be `extend`, but you'll figure it out

Or alternative variant - little longer, but 100% clear

Use 

```
OnlyofficeFileHelper::FileHelper.list_file_in_directory(path)
```

With this there is no any possibility to not know from there method coming